kf_lib_data_ingest/target_apis/kids_first_original_dataservice.py

```diff
@@ -0,0 +1,768 @@
+"""
+This module is translated into a etl.configuration.target_api_config.TargetAPIConfig
+object which is used by the load stage to populate instances of target model
+entities (i.e. participants, diagnoses, etc) with data from the extracted
+concepts before those instances are loaded into the target service (i.e Kids
+First Dataservice)
+
+See etl.configuration.target_api_config docstring for more details on the
+requirements for format and content.
+"""
+from d3b_utils.requests_retry import Session
+from requests import RequestException
+
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_lib_data_ingest.common.misc import str_to_obj
+
+
+def indexd_hashes(dictstr):
+    return {
+        k.lower().replace("-", ""): v for k, v in str_to_obj(dictstr).items()
+    }
+
+
+def join(*args):
+    """
+    Joins args using periods.
+    This is used when making compound unique keys from row data.
+
+    :return: `".".join([str(a) for a in args])`
+    :rtype: str
+    """
+    return ".".join([str(a) for a in args])
```

Comment:
I don't think `.` is a good delimiter choice. What if we have a unique key component that contains a float or string with a `.`? I think we had `-` before which probably also wasn't great. Let's pick something less likely to appear in a unique key component (e.g. `|`)

Reply:
√